cmd/ui/src/components/GraphEvents.tsx

```diff
@@ -26,47 +26,39 @@ import {
     resetCamera,
 } from 'src/ducks/graph/utils';
 import { bezier } from 'src/rendering/utils/bezier';
-import { getNodeRadius } from 'src/rendering/utils/utils';
+import { DRAG_DELAY, MOUSE_BUTTON_PRIMARY, getNodeRadius, preventAllDefaults } from 'src/rendering/utils/utils';
 import { sequentialLayout, standardLayout } from 'src/views/Explore/utils';
 
 export interface GraphEventProps {
-    onDoubleClickNode?: (id: string) => void;
     onClickNode?: (id: string) => void;
     onClickEdge?: (id: string, relatedFindingType?: string | null) => void;
     onClickStage?: () => void;
-    edgeReducer?: (edge: string, data: Attributes, graph: AbstractGraph) => Attributes;
     onRightClickNode?: (event: SigmaNodeEventPayload) => void;
     showNodeLabels?: boolean;
     showEdgeLabels?: boolean;
 }
 
 export const GraphEvents = forwardRef(function GraphEvents(
     {
-        onDoubleClickNode,
         onClickNode,
         onClickEdge,
         onClickStage,
         onRightClickNode,
-        edgeReducer,
         showNodeLabels = true,
         showEdgeLabels = true,
     }: GraphEventProps,
     ref
 ) {
-    const { selectedItem } = useExploreSelectedItem();
+    const { cancelHighlight, highlightedItem, selectedItem } = useExploreSelectedItem();
 
     const sigma = useSigma();
     const registerEvents = useRegisterEvents();
     const setSettings = useSetSettings();
 
-    const [hoveredNode, setHoveredNode] = useState<string | null>(null);
```

Comment:
BHE only sets a highlighted state when an item is clicked, not on hover. I removed the hover highlights to match the UX.